### DISCUSSION TITLE
Making the good decision

### DISCUSSION THREAD

**Discussion by mgrisole** [vercel/next.js#20555] [Help]
Hi guys, open-ended question here, what argument would you go with to compare Next VS Sapper?

I am posting this question in this section because it is real decision for our team to take and I feel like we might miss some key points in our thinking process.

I have been playing around with both of them, creating a simple page mimicking an e-commerce product list page. At the moment they tend to be really close in terms of performances but I was wondering if there might be some kind of scenario where one would outperform the other?

Does the heavier bundle size coming from React for Next, comes in handy when talking about real production scenarios?

Any thoughts or suggestions are welcome

**Comment by mnlfischer** [comment]
In my opinion performance is the second thing you should consider. Most important is the support and development of the project itself. I worked for companies who decided to use frameworks with no manpower behind.
Look at [t3](https://github.com/box/t3js) or [react-static](https://github.com/react-static/react-static). t3 is deprecated but a huge website was build on it and needs to maintained by the company I worked for. React Static will be the next one, I promise.
Comparing Next and Sapper I would choose Next with the experience I made. Vercel is a growing company and the release cycle of next is really fast. In these days I think it is the mostly established framework. Sapper is made by Svelte with a strong community, but no company behind.
Hope these informations can help you. Good luck with your decision!

**Reply by mgrisole** [reply]
Thank you very much for your feedback @mnlfischer, I share the same view regarding the potential risks of seing the technology being deprecated even though some big companies like Ikea are already using it. But it still feels like flipping a coin when it comes to choose between the team behind a technology, Google for instance didn't do so well in maintaining Angular.

~~At the moment I try to analyse the benefits of async data management libraries like [SWR](https://swr.vercel.app/) or [React Query](https://react-query.tanstack.com/) as I didn't find any real equivalence for Svelte, maybe it will come with the upcoming [Sveltekit](https://svelte.dev/blog/whats-the-deal-with-sveltekit) though.~~ (https://svelte-query.vercel.app/)
